main.c

```diff
@@ -20,14 +20,17 @@ int main(int argc, char *argv[])
     /* load with memory / and or setup peripherics */
 
     while(1) {
+        console_socket_check(&ge130);
         ge_clear(&ge130);
         ge_load(&ge130, &test_program, 1);
         ge_start(&ge130);
```

Comment:
I think we should separate these two ways of running the emulator, I think we can have an option to enable the console, and in this case, clear/load/start are only managed through the console, otherwise, a simpler clear/load/start like the one we have now is used.